contracts/pay.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity 0.5.11;
+
+contract pay {
```

Comment:
Add necessary comment for public contract and its APIs

Reply:
This is a unnecessary test file and removed.